src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,37 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE = DateTimeFormatter.ofPattern("dd.MM.yyyy");
 
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate localDateFrom = LocalDate.parse(dateFrom, DATE);
+        LocalDate localDateTo = LocalDate.parse(dateTo, DATE);
+        StringBuilder informationAboutSalary = new StringBuilder();
+        informationAboutSalary.append("Report for period ")
+                            .append(dateFrom)
+                            .append(" - ")
+                            .append(dateTo)
+                            .append("\n");
```

Comment:
```suggestion
        informationAboutSalary.append("Report for period ").append(dateFrom)
                            .append(" - ").append(dateTo);
```
.append("\n") краще перенести перед тим як додаєш ім'я, тоді не потрібн буде вкінці робити trim()